extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -66,7 +72,7 @@ public static void schedule(String thingUID, AstroThingHandler astroHandler, Job
             logger.error("{}", ex.getMessage(), ex);
         }
     }
-
+    
```

Comment:
5-6 days back I have set the SmartHome environment using Eclipse Oxygen RC release using Oomph Profile. I am using the current formatter. I will try to manually remove these lines.